catalog/rocks/cataloger.go

```diff
@@ -638,9 +641,18 @@ func (c *cataloger) Merge(ctx context.Context, repository string, leftBranch str
 	if err != nil {
 		return nil, err
 	}
+	count := make(map[catalog.DifferenceType]int)
+	if summary.Count != nil {
```

Comment:
Go's range works on nil, no need to check

Reply:
Cool, thanks!